frontend/discourse/admin/components/admin-welcome-banner-form.gjs

```diff
@@ -0,0 +1,518 @@
+              }}</select.Option>
+            <select.Option @value="below_site_header">{{i18n
+                "admin.config.welcome_banner.form.location.options.below_site_header"
+              }}</select.Option>
+          </field.Select>
+        </form.Field>
+
+        <form.Section
+          @title={{i18n "admin.config.welcome_banner.form.text_section.title"}}
+          @subtitle={{htmlSafe
+            (i18n
+              "admin.config.welcome_banner.form.text_section.variables"
+              username="%{username}"
+              site_name="%{site_name}"
+            )
+          }}
+        >
+
+          <form.Field
+            @name="headerNewMembers"
+            @title={{i18n
+              "admin.config.welcome_banner.form.header_new_members.label"
+            }}
+            @format="large"
+            @validation="required"
+            as |field|
+          >
+            <field.Input
+              placeholder={{i18n
+                "admin.config.welcome_banner.form.header_new_members.placeholder"
+              }}
+            />
+          </form.Field>
+
+          <form.Field
+            @name="headerLoggedInMembers"
+            @title={{i18n
+              "admin.config.welcome_banner.form.header_logged_in.label"
+            }}
+            @format="large"
+            @validation="required"
+            as |field|
+          >
+            <field.Input
+              placeholder={{i18n
+                "admin.config.welcome_banner.form.header_logged_in.placeholder"
+              }}
+            />
+          </form.Field>
+
+          <form.Field
+            @name="headerAnonymousMembers"
+            @title={{i18n
+              "admin.config.welcome_banner.form.header_anonymous.label"
+            }}
+            @format="large"
+            @validation="required"
+            as |field|
+          >
+            <field.Input
+              placeholder={{i18n
+                "admin.config.welcome_banner.form.header_anonymous.placeholder"
+              }}
+            />
+          </form.Field>
+
+          <form.Field
+            @name="subheaderLoggedInMembers"
+            @title={{i18n
+              "admin.config.welcome_banner.form.subheader_logged_in.label"
+            }}
+            @description={{i18n
+              "admin.config.welcome_banner.form.subheader_logged_in.description"
+            }}
+            @format="large"
+            as |field|
+          >
+            <field.Input />
+          </form.Field>
+
+          <form.Field
+            @name="subheaderAnonymousMembers"
+            @title={{i18n
+              "admin.config.welcome_banner.form.subheader_anonymous.label"
+            }}
+            @description={{i18n
+              "admin.config.welcome_banner.form.subheader_anonymous.description"
+            }}
+            @format="large"
+            as |field|
+          >
+            <field.Input />
+          </form.Field>
```

Comment:
Makes sense to switch to textarea for subheader, as it doesn't look pretty with the input:

<img width="485" height="256" alt="Image" src="https://github.com/user-attachments/assets/897f3f5b-8291-4fd7-8087-fcd52c55935f" />